docs/conf.py

```diff
@@ -35,9 +35,11 @@
     if isdir(join(exp, f))
 ]
 
-instr = "../instrumentation"
+instr = "../opentelemetry-python-contrib/instrumentation"
```

Comment:
Flagging this change because this is what was needed to pass the `docs` tests

Reply:
I think this PR will break the ReadTheDocs build 🙁 it needs to know to pull in the other repo somehow.

If you specify the other repo via git+https urls in the `docs-requirements.txt` file, it should actually work I think. You should be able to trigger a build of this PR against RTD somehow, I think there is an API endpoint to kick off a job

Reply:
I've removed the edit I made to the `docs/conf.py` folder. Turns out that we still have some instrumentation in the Core repo (i.e. `opentelemetry-instrumentation-opentracing-shim`) so it doesn't break after all. In reality, all its doing is adding that folder to the sys path so it can find packages there, so it should be find to leave it unchanged :+1:.

Reply:
Are the docs for contrib and core still both going to be on https://opentelemetry-python.readthedocs.io ?

Reply:
No, I remember we mentioned that at the last SIG and @lzchen said it would be fine if the docs are on 2 separate repos?

Reply:
Got it!